vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/treegrid/TreeGrid.java

```diff
@@ -1056,4 +1056,33 @@ public HierarchicalDataProvider<T, SerializablePredicate<T>> getDataProvider() {
     public void scrollToIndex(int rowIndex) {
         super.scrollToIndex(rowIndex);
     }
+
+    /**
```

Comment:
The JavaDoc of `scrollToIndex(int rowIndex)` should also be updated. It still mentions the issue we're fixing with this change.

Reply:
Updated the JavaDoc in the latest commits.